lib/slayer_rails/extensions/form.rb

```diff
@@ -28,9 +28,10 @@ def from_params(params, additional_params: {}, root_key: nil)
 
           def from_model(model)
             attr_names = attribute_set.map(&:name)
+            attr_hash = {}
 
-            attr_hash = attr_names.inject({}) do |n, hash|
-              hash[n] = model.public_send(n) if model.respond_to?(n)
+            attr_names.each do |attr_name|
```

Comment:
I think this could be:

```ruby
attr_hash = attr_names.select { |attr_name| model.respond_to?(attr_name) }
                                       .map { |attr_name|  [attr_name, model.public_send(attr_name)] }
                                       .to_h
```